skins/laika/src/components/pages/donation_confirmation/MembershipInfo.vue

```diff
@@ -3,9 +3,19 @@
         <div class="column is-half">
             <h2 class="title is-size-5">{{ $t( 'donation_confirmation_membership_call_to_action_title' ) }}</h2>
             <p>{{ $t( 'donation_confirmation_membership_call_to_action_text' ) }}</p>
-            <p><b-button type="is-primary is-main">{{ $t('donation_confirmation_membership_button') }}</b-button></p>
+            <p>
+                <a href="#">
+                    <b-button type="is-primary is-main has-margin-top-36">{{ $t('donation_confirmation_membership_button') }}</b-button>
+                </a>
+            </p>
+        </div>
+        <div class="column is-half">
+            <p>{{ $t( 'donation_confirmation_bottombox_membership_benefit_1' ) }}</p>
```

Comment:
Could you make this a list? The paragraph markup is not very semantic and looks funny when the lien is too long (text of the 2nd line on the same indentation level as the bullet point). With a list you can solve that.